README.md

```diff
@@ -308,6 +309,64 @@ az role assignment create --scope $He_ACR_Id --role acrpull --assignee $(eval $H
 - Instructions for [App Service](docs/AppService.md)
 - Instructions for [AKS](docs/aks/README.md#L233)
 
+## Smoke test setup
+
+Deploy [web validate](https://github.com/retaildevcrews/webvalidate) to drive consistent traffic to the App Service for monitoring and alerting.
+
+> For production deployments, you likely want to set --summary-minutes to 30 or 60
+
+```bash
+
+# Add Log Analytics extension
+az extension add -n log-analytics
+
+# create Log Analytics for the webv clients
+az monitor log-analytics workspace create -g $He_App_RG -l $He_Location -n $He_Name -o table
+
+# retrieve the Log Analytics ID
+export He_LogAnalyticsId=$(az monitor log-analytics workspace show -g $He_App_RG -n $He_Name --query customerId -o tsv)
+
+# add Log Analytics Keys to Key Vault (TODO: should secondary keys be added?)
+az keyvault secret set -o tsv --query name --vault-name $He_Name --name "LogAnalyticsPrimaryKey" --value $(az monitor log-analytics workspace get-shared-keys -g $He_App_RG -n $He_Name --query primarySharedKey -o tsv)
+az keyvault secret set -o tsv --query name --vault-name $He_Name --name "LogAnalyticsSecondaryKey" --value $(az monitor log-analytics workspace get-shared-keys -g $He_App_RG -n $He_Name --query secondarySharedKey -o tsv)
+
+# get the values
+export He_LogAnalytics_Primary_Key=$(az keyvault secret show -o tsv --query value --vault-name $He_Name --name LogAnalyticsPrimaryKey)
+export He_LogAnalytics_Secondary_Key=$(az keyvault secret show -o tsv --query value --vault-name $He_Name --name LogAnalyticsSecondaryKey)
+
+# save the environment variables
+./saveenv.sh -y
```

Comment:
Should I make a separate section for Log Analytics under the smoke test setup portion or the main one?

Reply:
I don't think so since they are interrelated. We can always tweak after we go through it a couple of times.